student-work/joe-wainer/api-final-project/post-reference-guide.md

```diff
@@ -0,0 +1,79 @@
+# POST Reference Guide
+POST is used to send commands to the server. The burger JSON object is included in the POST request which is sent to the kitchen. When an OK response is received, the order is printed in the kitchen so the kitchen staff can start cooking the order. 
+
+# POST/burgerMeal/burger Query Parameters
+
+| Name | Description | Default Value | Data Type | Required |
+| --- | --- | --- | --- | --- |
+| pattyType | The type of patty. Either “beef”, “chicken”, “vegetarian”. | beef | string | Yes |
```

Comment:
For all cases use straight quotes for the choices.
For example,
``beef`` vs "beef"